pkgs/development/libraries/edencommon/default.nix

```
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     glog
     folly
-    fmt_8
+    fmt_9
```

```suggestion
    folly.fmt
```

maybe too gross but `folly` has fmt and boost in pass thru. used by `fb303` but perhaps should be used by the whole fb ecosystem.

---

Right. I’m making stuff just get propagated on my update branch.